tests/article.test.js

```diff
@@ -0,0 +1,108 @@
+/*eslint-disable*/
+const expect = require('expect');
+const request = require('supertest');
+const {ObjectID} = require('mongodb');
+const defaults = require('superagent-defaults');
+const {app} = require('../server/app');
+const uuidv4 = require('uuid/v4');
+const Article = require('../server/DataBase/models/article');
+const {articles, populateArticles} = require('./articleseed');
+const {secretKeys} = require ('./../server/utilities/constants');
+const {users, populateUsers} = require('./seed');
+
+let articleOne;
```

Comment:
you can write this in one like let articleOne, articleTwo and so on

Reply:
Done